components/Layout.tsx

```diff
@@ -39,16 +39,13 @@ const ConversationLayout: React.FC = ({ children }) => {
   const router = useRouter()
   const recipientWalletAddress = router.query.recipientWalletAddr as string
 
-  const handleSubmit = useCallback(
-    async (address: string) => {
-      router.push(address ? `/dm/${address}` : '/dm/')
-    },
-    [router]
-  )
+  const handleSubmit = async (address: string) => {
+    router.push(address ? `/dm/${address}` : '/dm/')
+  }
 
-  const handleBackArrowClick = useCallback(() => {
+  const handleBackArrowClick = () => {
```

Comment:
It seems like removing the `useCallback` here will force a rerender of the `BackArrow` on every render of this component.

Reply:
I had added the useCallback back here